platform/contracts/admin/src/contracts/protocol/mod.rs

```diff
@@ -13,3 +16,10 @@ pub struct Protocol<T> {
     pub oracle: T,
     pub profit: T,
 }
+
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[serde(rename_all = "snake_case", deny_unknown_fields)]
+pub struct ProtocolWithNetworkName {
```

Comment:
this type should be named `Protocol` whereas the former one - `ProtocolTemplate`